lib/govuk_test.rb

```diff
@@ -11,10 +11,16 @@ def self.configure(options = {})
     chrome_options << "--window-size=#{options[:window_size]}" if options[:window_size]
 
     if ENV['GOVUK_TEST_USE_SYSTEM_CHROMEDRIVER']
```

Comment:
We can problem change this to an `unless` and avoid the assignment, unless @cbaines it's doing something unexpected?

Reply:
I'm not sure I follow what the question is?

Reply:
So I don't think `File.which` actually does anything. By default it'll just run `chromedriver`, so the only difference this makes is that we run it on an absolute path; either way, the binary has to be on the PATH.

Reply:
Yeah, I was mostly copying what the `chromedriver-helper` gem was doing. What you say sounds correct.